findimports.py

```diff
@@ -1017,12 +1111,18 @@ def main(argv=None):
                         ignore_stdlib_modules=args.ignore_stdlib)
     if args.write_cache:
         g.writeCache(args.write_cache)
+
     if args.condense_to_packages:
-        g = g.packageGraph(args.packagelevel)
+        g = g.packageGraph(args.packagelevel, externals_only=False)
+    elif args.condense_to_packages_externals:
+        g = g.packageGraph(args.packagelevel, externals_only=True)
+
     if args.collapse_tests:
         g = g.collapseTests()
     if args.collapse_cycles:
         g = g.collapseCycles()
+    if args.rmprefix is not None:
+        g = g.removePrefixes(args.rmprefix)
```

Comment:
I fear that this may be a bit of a foot gun.  My reading of the code suggests that `--rmprefix a` will not only change `a.x` to `x`, but also `ab.x` to `b.x`, which doesn't seem right to me.

Reply:
Agreed.
```python
re.sub(r'(({})\.)*'.format('|'.join(prefixes)), '', string)
```
This ok?

Reply:
Either that or `reg_cmp = re.compile(r'^(({})\.)?'.format('|'.join(prefixes)))`

Maybe this is better as chain OR elimination runs risk of devouring too much.